### PR TITLE
Fd 1065 rejected messages

### PR DESCRIPTION
**Pull request by factom-clay**

### CODE REVIEW

**Comment by carryforward** [comment]
FD-1065 is an epic ticket.  should this branch be named some other FD number?

File: state/missingMsgTracking.go [hunk 1]
```diff
@@ -72,7 +83,7 @@ func (m *RecentMessage) Add(msg interfaces.IMsg) {
 			delete(m.MsgsMap.Msgs, oldHash)
 
 			// Add the new message
-			hash := msg.GetHash().Fixed()
+			hash := msg.GetMsgHash().Fixed()
```
Comment (stackdump) [inline state/missingMsgTracking.go]: This was a bugfix correct?

File: state/processList.go [hunk 2]
```diff
@@ -750,207 +749,218 @@ func (p *ProcessList) decodeState(Syncing bool, DBSig bool, EOM bool, DBSigDone
 
 }
 
-var extraDebug bool = false
+var extraDebug bool = true
```
Comment (stackdump) [inline state/processList.go]: was it intentional to leave this set to true?

File: state/missingMsgTracking.go (hunk 1 above)
Comment (factom-clay) [reply state/missingMsgTracking.go]: yes

**Comment by factom-clay** [comment]
Closing wrong branch name see FD-1129

**Pull request closed without merging**